### ISSUE TITLE
dotnet with CPLEX: not linked or license not found

### ISSUE THREAD

**Issue by jjaneto** [google/or-tools#3475]
**What version of OR-Tools and what language are you using?**
Version: stable/v9.4
Language: C#

**Which solver are you using (e.g. CP-SAT, Routing Solver, GLOP, BOP, Gurobi)**

CPLEX 20.10

**What operating system (Linux, Windows, ...) and version?**

Windows 11

**What did you do?**

Right after cloning the repository:

1. Changed `Makefile.cpp.mk` adding `USE_CPLEX ?= ON` and providing the `-DCPLEX_ROOT` option in the `third_party` target recipe.
2. Executed: ` tools\make third_party`
3. Executed then: ` tools\make dotnet`
4. Created a console application project from scratch in VS 2022.
5. Installed the two nuget packages located in the `.\or-tools\build_make\dotnet\packages` into my project.

**What did you expect to see**

Execute with success the [simple MIP example](https://github.com/google/or-tools/blob/stable/ortools/linear_solver/samples/SimpleMipProgram.cs).

**What did you see instead?**

An error. The console outputs: 

```
WARNING: Logging before InitGoogleLogging() is written to STDERR
W0926 16:52:33.024684 30456 linear_solver.cc:625] Support for cplex not linked in, or the license was not found.
```

**Anything else we should know about your project / environment**

I thought that this could be an error involving not supported CPLEX versions. After seeing [this answer](https://github.com/google/or-tools/issues/3252#issuecomment-1169677799), I changed the FindCPLEX.cmake file right after the `elseif(MSVC)` line to: 

`set_target_properties(CPLEX::CPLEX PROPERTIES
      IMPORTED_LOCATION "${CPLEX_ROOT}/cplex/lib/x64_windows_msvc14/stat_mda/cplex2010.lib")`

which was somehow `cplex12100.lib` before.

**Comment by lperron** [comment]
Can you check it does not uses the package from nuget ?

**Comment by jjaneto** [comment]
> Can you check it does not uses the package from nuget ?

Thanks for the reply.

I'm using the nuget packages generated from the compilation process. 
Besides, I updated the section "*What did you do*" with steps 4. and 5. I hope this clarifies! 

PS: a friend of mine reproduced steps 4. and 5. **in his computer**, with the .nupkgs generated in **my** local, and it worked with CPLEX 12.7!

**Comment by Mizux** [comment]
@jjaneto in your cplex install dir what is the name of the "`cplex*.lib`" (and its relative location from the cplex root dir installation) ?

**Comment by jjaneto** [comment]
@Mizux the name of the lib is `cplex2010.lib`; I installed in my local with the default settings. The relative path to this library is `IBM\ILOG\CPLEX_Studio201\cplex\lib\cplex2010.lib`. 

I also checked for any error involving environment variables, but could not find evidences for it.

I reproduced again the steps of the original post, changing the `FindCPLEX.cmake` file according your suggestion (editing to `cplex*.lib`), but the outcome I got the same error from before!
